components/operator/apis/stack/v1beta3/gateway_types.go

```diff
@@ -11,4 +11,7 @@ type GatewaySpec struct {
 
 	// +optional
 	EnableAuditPlugin *bool `json:"enableAuditPlugin,omitempty"`
+
+	// +optional
```

Comment:
Seems to be useless, you already know the endpoint no?

Reply:
No i don't know the endpoint. 

In Nats case
I can guess the host name. 
But I need to hardcode monitoring port somewhere.